src/Page/Auth.elm

```diff
@@ -0,0 +1,698 @@
+            let
+                newSession =
+                    Session.logout session
+                        |> Session.notifyInfo "Vous n'avez plus accès au détail des impacts" ""
+            in
+            ( model
+            , newSession
+            , Cmd.batch
+                [ newSession.store |> Session.serializeStore |> Ports.saveStore
+                , logout
+                ]
+            )
+
+        TokenEmailSent response ->
+            ( { model | response = Result.toMaybe response }
+            , case response of
+                Ok _ ->
+                    session
+
+                Err _ ->
+                    session
+                        |> Session.notifyError "Erreur lors de la connexion" ""
+            , Cmd.none
+            )
+
+        UpdateForm newModel ->
+            ( newModel
+            , session
+            , Cmd.none
+            )
+
+
+view : Session -> Model -> ( String, List (Html Msg) )
+view session model =
+    ( "Authentification"
+    , [ Container.centered [ class "pb-5" ]
+            [ h1 [ class "mb-3" ]
+                [ text <|
+                    if Session.isAuthenticated session then
+                        "Compte"
+
+                    else
+                        "Connexion / Inscription"
+                ]
+            , div [ class "row justify-content-center" ]
+                [ if Session.isAuthenticated session then
+                    div [ class "row d-flex justify-content-center" ]
+                        [ if model.loggedIn then
+                            p [ class "text-center" ]
+                                [ text "Vous avez maintenant accès au détail des impacts, à utiliser conformément aux "
+                                , a [ href Env.gitbookUrl ] [ text "conditions" ]
+                                , text "."
+                                ]
+
+                          else
+                            text ""
+                        , viewAccount model
+                        , button
+                            [ onClick Logout
+                            , class "btn btn-link mb-3"
+                            ]
+                            [ text "Déconnexion" ]
+                        ]
+
+                  else
+                    div [ class "row d-flex justify-content-center" ]
+                        [ div [ class "col-xl-12" ]
+                            [ Alert.simple
+                                { level = Alert.Info
+                                , close = Nothing
+                                , title = Nothing
+                                , content =
+                                    [ div [ class "fs-7" ]
+                                        [ """Pour avoir accès au détail des impacts, il est nécessaire de s'enregistrer et
+                                    valider que vous êtes Français, et que vous n'utiliserez pas ces données à des fins
+                                    commerciales."""
```

Comment:
Is this really the final text we want to render or should we ask for validation?

Reply:
Thx I noted it down to ask